modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -1026,6 +1028,56 @@ public void testPutObjectDefinition() throws Exception {
 		_objectDefinitionLocalService.deleteObjectDefinition(
 			randomModifiableSystemObjectDefinition.getId());
 
+		// Modifiable system object definition different language ID
+
+		randomModifiableSystemObjectDefinition =
+			objectDefinitionResource.postObjectDefinition(
+				_randomModifiableSystemObjectDefinition());
+
+		objectDefinitionDefaultLanguageId = "pt_BR";
+
+		siteDefaultLanguageId1 = LanguageUtil.getLanguageId(
+			LocaleUtil.getSiteDefault());
+
+		Assert.assertNotEquals(
+			objectDefinitionDefaultLanguageId, siteDefaultLanguageId1);
+
+		randomModifiableSystemObjectDefinition.setDefaultLanguageId(
+			objectDefinitionDefaultLanguageId);
+		randomModifiableSystemObjectDefinition.setLabel(
+			MapUtil.fromArray(
+				objectDefinitionDefaultLanguageId,
+				RandomTestUtil.randomString()));
+
+		String siteDefaultLanguageId2 = "es_ES";
```

Comment:
As this is not the site default locale, you could use something like `Locale esLocale = LocaleUtil.fromLanguageId("es_ES");`. But it's better to get rid of this variable, so please inline `LocaleUtil.fromLanguageId("es_ES")` in your usages.